test/functionalTest/cases/2507_csub_metadata_field/previous_value_with_compounds.test

```diff
@@ -279,13 +279,13 @@ Fiware-Correlator: REGEX([0-9a-f\-]{36})
             "A": {
                 "metadata": {
                     "previousValue": {
-                        "toplevel": [
```

Comment:
This is funny! :)

It seems we have a bug in compound metadata rendeering logic. With the new implementaiton of the renders, now seems to be fixed and this .test has been modified to align with it.

Reply:
Fix mentioned in CNR

NTC (I guess)

Reply:
NTC